R/data_maggie.R

```diff
@@ -0,0 +1,464 @@
+# TODO: some variables are not defined in articles or datasets-- is there another source I can check?
+#   For "Complete history of the NBA" I wasn't sure about the meaning of the variable "neutral" (consulted outside sources)
+#   For "What the world think of trump" I couldn't figure out dk_refused-- maybe percent of respondents who refused to answer?
+#   For the "How popular is Donald Trump" first dataset I couldn't figure out "multiversions" variable (maybe if multiple versions of the poll were sent out?) and second dataset I couldn't figure out what the models were based on (it seemed like it should be one model per poll but there were far fewer models than polls)
+
+#' The Complete History Of The NBA
+#' 2017-18 NBA Predictions
+#' 
+#' The raw data behind the story
+#' "The Complete History Of The NBA"
+#' \url{https://projects.fivethirtyeight.com/complete-history-of-the-nba/} 
+#' And our "2017-18 NBA Predictions"
+#'\url{https://projects.fivethirtyeight.com/2018-nba-predictions/}
+#'
+#'@format A data frame with 67,012 rows representing games and 20 variables:
+#'\describe{
+#'  \item{date}{Date}
+#'  \item{season}{Season year, 1947-2018}
+#'  \item{neutral}{TRUE if the game was played on neutral territory, FALSE if not}
+#'  \item{playoff}{TRUE if the game was a playoff game, FALSE if not}
+#'  \item{team1}{The name of one participating team}
+#'  \item{team2}{The name of the other participating team}
+#'  \item{elo1_pre}{Team 1's Elo rating before the game}
+#'  \item{elo2_pre}{Team 2's Elo rating before the game}
+#'  \item{elo_prob1}{Team 1's probability of winning based on Elo rating}
+#'  \item{elo_prob2}{Team 2's probability of winning based on Elo rating}
+#'  \item{elo1_post}{Team 1's Elo rating after the game}
+#'  \item{elo2_post}{Team 2's Elo rating after the game}
+#'  \item{carmelo1_pre}{Team 1's CARMELO rating before the game}
+#'  \item{carmelo2_pre}{Team 2's CARMELO rating before the game}
+#'  \item{carmelo1_post}{Team 1's CARMELO rating after the game}
+#'  \item{carmelo2_post}{Team 2's CARMELO rating after the game}
+#'  \item{carmelo_prob1}{Team 1's probability winning based on CARMELO rating}
+#'  \item{carmelo_prob2}{Team 2's probability of winning based on CARMELO rating}
+#'  \item{score1}{Points scored by Team 1}
+#'  \item{score2}{Points scored by Team 2}
+#'}  
+#'@source See \url{https://projects.fivethirtyeight.com/nba-model/nba_elo.csv}
```

Comment:
A couple of things

* You need to add the dataframe name in quotation marks for the roxygnen help files to generate in `/man`. so for example here, you need to add `"nba_carmelo"` to line 39 for the help file to generate. Please do this for all data frames below.
* For style/legibility purposes, can ensure there is always a single space after all `#'` markings? See `data_albert.R` as an example. A replace all of `#'` with `#' ` in Rstudio should do the trick